### ISSUE TITLE
Rename Bot.graph

### ISSUE THREAD

**Issue by Debilski** [ASPP/pelita#808]
I suggest we rename `Bot.graph` that was introduced in #792 to something that mirrors its purpose and contents better instead of giving it a name that merely reflects its data type. :)

`Bot.path`, `Bot.way`, `Bot.free`, `Bot.squares`?

**Comment by otizonaizit** [comment]
Yes, it's worth talking about it an maybe find a better name. The good thing about `graph` is that it helps understanding that it is a special thing which comes from `networkx` and that you can apply graph's algorithms on, like for example `shortest_path`. This alone in my opinion would discourage naming it `path` or `way`, because those are actually most probably the result of applying shortest path on the graph to get exactly that: a way or a path towards something. A group this semester at the TU even copied the graph and modified it to remove "dangerous" coordinates from it, so that they could use shortest_path on it in a safe way. I think it is important that the names conveys the "speciality" of the thing.

About the other names:

Given that bot.walls and bot.food are sequences, it would be confusing having bot.free to be something completely different (and also confusing because of "is a square free if it contains food or a bot sits on it?"-type of questions). Bot.squares is more generic, but you would still be a surprise that it is not just a list of coordinates...

So maybe bot.maze_graph or bot.maze?
